lib/db.c

```diff
@@ -4464,6 +4464,10 @@ grn_table_setoperation(grn_ctx *ctx, grn_obj *table1, grn_obj *table2, grn_obj *
   uint32_t value_size = 0;
   uint32_t key_size = 0;
   grn_bool have_subrec;
+  if (!table1 || !table2) {
+    WARN(GRN_INVALID_ARGUMENT, "table is null");
```

Comment:
We should provide more meaningful error message like: https://github.com/groonga/groonga/blob/master/lib/db.c#L313